railgun/internal/manager/config.go

```diff
@@ -79,8 +79,10 @@ type Config struct {
 	// Admission Webhook server config
 	AdmissionServer admission.ServerConfig
 
-	// Performance monitoring
-	EnableProfiling bool
+	// Diagnostics and performance
+	EnableProfiling     bool
+	EnableConfigDumps   bool
+	DumpSensitiveConfig bool
```

Comment:
do we need DumpSensitiveConfig ? 
Isnt if EnableConfigDumps == true, means that the controller is able to dump data? we just need a flag to the component to dump data, less duplicates.

Reply:
It's to make the default output something that can be safely sent to us for issue reports, since we don't want users to accidentally publish their private keys.

The actual content of the redacted fields usually isn't interesting, since we just copy them verbatim from Secrets.